packages/DatePicker/src/components/DateInput/DateInput.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+
+const propTypes = {
+  /** a11yText on the input. */
+  a11yText: PropTypes.string,
```

Comment:
why do we need this @allison-c  couldn't we just extract the value from the input?  cc:
@mikrotron

Reply:
It's nice to provide the ability of customized aria-label.

Reply:
I'm not sure it's intended to be used since this comonent isn't fully implemented yet, but since it will render an `<input>` and all inputs need a label, either a `<label>` (which we'd have to visually hide) or an `aria-label` attribute (which is simpler), this `a11yText` prop seems like an easy way to handle that.

Reply:
`a11yText` will be passed into `<Input />` component. I can actually remove it here, and if developers still want it, they can still pass it into `<DatePicker.Input />`, it will still work.